pybricksdev/cli/__init__.py

```diff
@@ -160,6 +161,13 @@ def add_parser(self, subparsers: argparse._SubParsersAction):
             default=True,
         )
 
+        parser.add_argument(
+            "--resend",
```

Comment:
"resend" is only one of possibly more menu options. I would be tempted to call this option `--stay-connected` or something like that.